lib/brains/views/method.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from lib.brains.client import get_client
+from lib.brains.errors import BraintreeResultError
+from lib.brains.forms import PaymentMethodForm
+from lib.brains.serializers import (
+    PaymentMethodSerializer as BraintreeSerializer)
+from lib.buyers.models import PaymentMethod
+from lib.buyers.serializers import PaymentMethodSerializer
+from lib.transactions.constants import PROVIDER_BRAINTREE
+from solitude.constants import PAYMENT_METHOD_CARD
+from solitude.logger import getLogger
+
+
+log = getLogger('s.brains')
+
```

Comment:
ok, I understand it better. I guess to keep them fully separate we'd want to have the client post first to ``braintree/paymethod`` then secondly to ``generic/paymethod`` but that would be an extra post and extra failure management for the client. In other words, now it seems awkward to me that the braintree endpoint creates a generic payment method behind the scenes ;) but I guess it's good encapsulation.